circuit_knitting/cutting/__init__.py

```diff
@@ -57,7 +57,7 @@
     qpd.generate_qpd_weights
     qpd.generate_qpd_samples
     qpd.decompose_qpd_instructions
-    qpd.supported_gates
+    qpd.explicitly_supported_gates
```

Comment:
I wanted to suggest `optimally_supported_gates`, but I can't convince myself I like it more. I think they are all optimal, at least?

Reply:
Yeah, they are all optimal -- even the ones for which we do not have explicit support.

Another option could be to get rid of this function -- or at least to make it private.  The reason it was introduced in #277 was to support #278, but now that _all_ two-qubit gates can be cut, it's no longer necessary.  It might be nice to have something like this if we one day support some (but not all) 3-qubit gates (#258), which could be a case for keeping it but prefixing it with an underscore.  This way, we'd at least have a use case in mind (again) before committing to support it.

By the way, all _essential_ tests (i.e., all but a few specific ones) work if we take this PR and remove all the explicit gate support, i.e., if we apply the further patch:

```diff
diff --git a/circuit_knitting/cutting/qpd/qpd.py b/circuit_knitting/cutting/qpd/qpd.py
index 1e5be49..c6974a8 100644
--- a/circuit_knitting/cutting/qpd/qpd.py
+++ b/circuit_knitting/cutting/qpd/qpd.py
@@ -573,13 +573,6 @@ def qpdbasis_from_gate(gate: Gate) -> QPDBasis:
         ValueError: Cannot decompose gate with unbound parameters.
         ValueError: ``to_matrix`` conversion of two-qubit gate failed.
     """
-    try:
-        f = _qpdbasis_from_gate_funcs[gate.name]
-    except KeyError:
-        pass
-    else:
-        return f(gate)
-
     if isinstance(gate, Gate) and gate.num_qubits == 2:
         try:
             mat = gate.to_matrix()
```

so explicit support doesn't mean a ton anyway.

Reply:
The more I thought about it, the more I felt like the most reasonable thing to do is to remove this function from the public API for now, hence my change in 8e95c4acdc369b8535682521da77d6cb810822b6.